### ISSUE TITLE
Dropdown: Focused element in dropdown list is not visible in high contrast

### ISSUE THREAD

**Issue by b-priyank** [OfficeDev/office-ui-fabric-react#1273]
### Bug Report

- __Package version(s)__: office-ui-fabric-react 1.14.2
- __Browser and OS versions__: EDGE, Windows 10

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)
No
Requested priority: (Blocking, High, Normal, Low) 
Normal
#### Describe the issue:

When in high contrast mode, Dropdown list focused item is black and not visible.

![image](https://cloud.githubusercontent.com/assets/9213358/23948058/58df5e0c-09a7-11e7-9b59-3cb63495caed.png)

#### Actual behavior:

When in high contrast mode, Dropdown list focused item is black and not visible.

#### Expected behavior:

When in high contrast mode, Dropdown list focused item background color and font color should be such that they are visible.

**Comment by sulabh-msft** [comment]
@gitjain : Please let me know if there is any update on this issue

**Comment by mdahamiwal** [comment]
Here is some background on this:
Its an issue with Edge taking control of the high contrast styling and providing a backplate. It happens when we apply our own high contrast styling without using (–ms-high-contrast-adjust: none). 
We had a followup discussion as part of https://microsoft.visualstudio.com/OS/_workitems?id=10464048&_a=edit and @mikewheaton created another https://github.com/OfficeDev/office-ui-fabric-core/issues/907 to track it on fabric side. Few changed were done in ms-fabric wrapper but this still seems to be an issue.
